### DISCUSSION TITLE
How, exactly, do I uninstall tabby?

### DISCUSSION THREAD

**Discussion by Decoherent** [Eugeny/tabby#10054] [Q&A]
This is one of the stupidest questions I've had to ask in a long time. When I found & was playing with tabby, it was really late at night, and I don't remember what I did with it. I installed *something*, clearly, because I have a couple context entries. The only file I have in my Downloads directory is the portable version, and I'd like to get rid of it and just install the app normally. Generally, I download "portable" versions of apps to test them out before letting it at my computer, but the packaging software and I disagree :) 

I have "Paste path into tabby" (which piqued my interest) and "Open tabby here" items in my context menu. However, there is nothing in the Start menu, the new-ish Settings menu, nor in the proper 'Uninstall a program". The app appears to be installed in "C:\portapps\tabby-portable", but just deleting the directory makes everything freak out (as it should) and doesn't remove he context entries.

So, what can I do?
Windows 10 Build 19045, tabby-1.0.215-portable

**Comment by Decoherent** [comment]
For posterity's sake:

In a Debian window, I let find clear out files & directories:
```
$ find / -type f -iname "*tabby*" -delete
$ find /mnt/c -type d -not -path "/mnt/c/\$Recycle.Bin/*" -iname "*tabby*" -delete
```

Then, I just let search run through the registry and removed the keys it added. I believe I found 5, and then some that pointed to caches. Nothings caught on fire yet. YMMV!

**Reply by Eugeny** [reply]
You can cleanly remove the context menu entries by disabling "Shell integration" from the app's settings.
